src/program/services/streaming/stream_connection.py

```
@@ -0,0 +1,90 @@
+    @start_position.setter
+    def start_position(self, value: int | None) -> None:
+        """Set the start position in the stream."""
+
+        if value is None:
+            if hasattr(self, "_start_position"):
+                del self._start_position
+
+            return
+
+        if value < 0:
+            raise ValueError("Start position cannot be negative")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)